modules/SofaExporter/STLExporter.h

```diff
@@ -56,15 +56,16 @@ class SOFA_EXPORTER_API STLExporter : public core::objectmodel::BaseObject
     unsigned int maxStep;
 
     std::ofstream* outfile;
-
-    void writeSTL();
-    void writeSTLBinary();
     
     int nbFiles;
 
 public:
+
+    void writeSTL();
+    void writeSTLBinary();
+
     sofa::core::objectmodel::DataFileName stlFilename;
-    Data<bool> m_fileFormat;      //0 for Ascii Formats, 1 for Binary File Format
```

Comment:
Though I agree to name this var correctly, I don't see its relevance to this particular PR ? But I guess it's ok to let this change in this PR, doing a separate PR just for it would be a useless overhead...